test/blackbox/delete_images.bats

```diff
@@ -0,0 +1,127 @@
+load helpers_zot
```

Comment:
Add Makefile target + ci check

Reply:
Will consolidate in https://github.com/project-zot/zot/issues/1743